hangman/lib/hangman/interface.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Hangman.Interface do
```

Comment:
So what you're doing here is totally decoupling the human player from this server.